backend/backend/production_settings.py

```diff
@@ -12,29 +12,23 @@
 
 import os
 from pathlib import Path
+from .settings import *
+import sys
+import mimetypes
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
+mimetypes.add_type("text/css", ".css", True)
 
+TESTING = sys.argv[1:2] == ['test']
+
+SECRET_KEY = os.environ['SECRET_KEY'] if not TESTING else "test"
 
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
-SECRET_KEY = os.environ['SECRET_KEY']
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = False
 
-
-ASYNC_GENERATION = True
-
 # Security settings
-ALLOWED_HOSTS = [
-    'commanderspellbook.com',
-    'localhost',
-    '127.0.0.1',
-    'backend.commanderspellbook.com',
-]
+ALLOWED_HOSTS = ['*']
```

Comment:
This is not a very safe idea because of the presence of classic HTML forms in the Django admin dashboard.
If we want to support custom hosting we could take the allowed host from environment variables? Or else we could simply hardcode it.

This article I found with google explains the need: https://www.divio.com/blog/django-allowed-hosts-explained/

Reply:
True, it is safer. I was getting some weird errors with it before, let me see if I can find a better solution.